api_app/views.py

```diff
@@ -0,0 +1,154 @@
+    queryset = Cover.objects.all()
+    serializer_class = CoverSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class ScreenshotRestView(viewsets.ModelViewSet):
+    queryset = Screenshot.objects.all()
+    serializer_class = ScreenshotSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class MustRestView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        queryset = Must.objects.all()
+        if len(queryset) != 0:
+            serializer = MustSerializer(queryset, many=True)
+            return Response(serializer.data)
+        return Response([])
+
+    def post(self, request):
+        game_id = int(request.data['game_id'])
+        user = request.user.id
+        try:
+            Must.objects.get_or_create(game=Game.objects.get(id=game_id), user=user)
+        except Game.DoesNotExist:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+        return Response(status=status.HTTP_201_CREATED)
+
+
+class MustObjectRestView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get_object(self, pk):
+        try:
+            return Must.objects.get(pk=pk)
+        except Must.DoesNotExist:
+            raise Http404
+
+    def get(self, request, pk):
+        must = self.get_object(pk)
+        serializer = MustSerializer(must)
+        return Response(serializer.data)
+
+    def put(self, request, pk):
+        must = self.get_object(pk)
+        data = {k: v for k, v in request.data.items()}
+        data['user'] = request.user.id
+        serializer = MustSerializer(must, data=data)
+        if serializer.is_valid():
+            try:
+                Must.objects.get(game=Game.objects.get(id=int(data['game'])),
+                                 user=UserModel.objects.get(id=int(data['user'])))
+                return Response(status=status.HTTP_409_CONFLICT)
+            except Must.DoesNotExist:
+                serializer.save()
+                return Response(serializer.data)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def delete(self, request, pk):
+        self.get_object(pk).delete(force=True)
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class Registration(APIView):
+    def post(self, request):
+        serializer = UserSerializer(data=request.data)
+        if serializer.is_valid():
+            user = serializer.save()
+            user.is_active = False
+            user.save()
+            send_email(request, user, True)
+            return Response('Verification email has been sent to you mail')
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class SendEmailAgain(APIView):
+    def post(self, request):
+        email = request.data.get('email')
+        if email:
+            try:
+                user = UserModel.objects.get(email=email)
+            except UserModel.DoesNotExist:
+                pass
```

Comment:
It would be good to return here smth like
`return Response('Bad email', status=status.HTTP_400_BAD_REQUEST)`